__tests__/integration.js

```diff
@@ -82,6 +82,22 @@ test('--mutex network', async () => {
   ]);
 });
 
+test('--cwd option', async () => {
+  const cwd = await makeTemp();
+  const cacheFolder = path.join(cwd, '.cache');
+
+  const subdir = path.join(cwd, 'a/b/c/d/e/f/g/h/i');
```

Comment:
Since you're using `path.join` I'd recommend writing this as `path.join(cwd, 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i')` for cross-platform compatibility. You can even do `path.join(cwd, ...Array.from('abcdefghi'))` if you wanna look cool :P